src/js/index.js

```diff
@@ -32,10 +32,10 @@ class Livewire {
     }
 
     activatePolyfills() {
-        ArrayFlat();
-        ArrayFrom();
-        ArrayIncludes();
-        ElementGetAttributeNames();
+        ArrayFlat()
```

Comment:
Ive removed the semi colons here, it seemed inconsistent with the rest of the files, i couldn't figure out what the intention here was?

Maybe `eslint` could be put in place for the package (happy to look into this as a future pr if i have time).

Reply:
Yeah, possibly, but you're right - the semicolons should be gone. I think I actually removed them since this PR.